recipes/nmma/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "nmma" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nmma-{{ version }}.tar.gz
+  sha256: 9394a212c06a60e06181b2f0b78e8015d41280aac89dbc6ab19156c6f09ab3fa
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

You failure on Windows was probably the lack of entrypoints.

```suggestion
  noarch: python
```

---

@ocefpaf I did see that bilby did not build on windows, which is why I ended up avoiding trying to use it.